maintainers/maintainer-list.nix

```diff
@@ -18761,6 +18761,11 @@
     githubId = 1595132;
     name = "Kranium Gikos Mendoza";
   };
+  woojiq = {
+    github = "woojiq";
+    githubId = 122799969;
+    name = "Woojiq";
+  };
```

Comment:
This should be added in it's own commit, as the first commit in this PR, with the summary `maintainers: add <handle>`

Reply:
Done. Btw, thanks for looking at this PR.